src/routes/search/+page.svelte

```diff
@@ -4,9 +4,14 @@
     import { refs } from '$lib/data/stores';
     import { query } from '$lib/scripts/query';
     import { postQueries, queries } from 'proskomma-tools';
+
```

Comment:
Why is this file changed?  It seems to just have extra newlines added without any real changes.

Reply:
I can't explain why, could it be when I first used prettier it could've added those lines?

Reply:
Prettier doesn't normally add blank lines.  Please remove this file from the PR.

Reply:
Deleted file

Reply:
I undid this realizing I did it incorrectly. reverted file back